OWNERS_ALIASES

```diff
@@ -170,14 +170,18 @@ aliases:
     - smarterclayton
     - soltysh
   sig-testing-reviewers:
+    - bentheelder
+    - cblecker
     - fejta
+    - krzyzacy
     - ixdy
-    - rmmh
     - spiffxp
   sig-testing-approvers:
```

Comment:
Shouldn't we add more here?

/lgtm
/hold

Reply:
move all top level approvers here?

Reply:
SGTM

Reply:
Actually I'll keep it like this, given other test-infra folks might not want to get spammed on those k8s bash 🔥. If you wish to have your name added here feel free to PR :-) (@amwat @cjwagner @stevekuznetsov @Katharine @mithrav @michelle192837 )